content-security-policy/support/echo-embedding-csp.py

```diff
@@ -0,0 +1,11 @@
+import json
+def main(request, response):
+    header = request.headers.get("Embedding-CSP");
+    message = {}
+    message['embedding_csp'] = header if header else None
+    return [("Content-Type", "text/html")], '''
```

Comment:
I think the tests would be slightly different. Here we expect to get Embedding-CSP (from say a browser) and if so, post a message.
In the other case, depending on what browser sent, we decide to send or not Allow-CSP-From header. But I am working on that PR now!